M/MUSCLE/build_tarballs.jl

```diff
@@ -11,14 +11,21 @@ sources = [
 # Bash recipe for building across all platforms
 script = raw"""
 cd ${WORKSPACE}/srcdir/muscle/src/
-make -j${nproc} CXXFLAGS="-O3 -fopenmp"
-install -Dvm 755 "Linux/muscle" "${bindir}/muscle"
+
+if [[ "${target}" == *-apple-* ]]; then
+    make -j${nproc} CXX="clang++" CXXFLAGS="-O3 -fopenmp"
+else
+    make -j${nproc} CXXFLAGS="-O3 -fopenmp"
+fi
+
+OS_NAME=`uname`
+install -Dvm 755 "${OS_NAME}/muscle" "${bindir}/muscle"
```

Comment:
Can also do
```suggestion
install -Dvm 755 "$(uname)/muscle" "${bindir}/muscle"
```
if you don't need the variable for anything else